datacats/environment.py

```diff
@@ -255,9 +261,9 @@ def data_complete(self):
         """
         Return True if all the expected datadir files are present
         """
-        if (not isdir(self.datadir + '/files')
-                or not isdir(self.datadir + '/run')
-                or not isdir(self.datadir + '/search')):
+        if (not isdir(self.datadir + '/files') or
+                not isdir(self.datadir + '/run') or
+                not isdir(self.datadir + '/search')):
```

Comment:
oh, I also don't like this. the "or" in this continuation is very important for understanding the logic so I prefer it at the beginning of the line near the rest of the logical operators.